<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Кофе</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>761</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add_edit_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>152</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить / изменить запись</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
